overrides.mk

```diff
@@ -3,9 +3,9 @@
 #
 
 # DEFAULT values
-# ubi8/ubi-micro:8.8-1
-DEFAULT_BASEIMAGE="registry.access.redhat.com/ubi8/ubi-micro:8.8-1"
-DEFAULT_GOVERSION="1.20"
+# ubi9/ubi-micro:latest
+DEFAULT_BASEIMAGE="registry.access.redhat.com/ubi9/ubi-micro:latest"
```

Comment:
We should be using 9.2-13 as there is a possibility of another new version being used after code freeze date if '_latest_' tag is used.

Reply:
yes you are correct. Updated it to 9.2-13 thanks for pointing it out. Please review it again.